app/submit/index.js

```diff
@@ -0,0 +1,15 @@
+const db = require('../data')
+
+async function createAgreement (message) {
```

Comment:
You could just pass the body of the message to this function rather than the full service bus message.

Would make it simpler if the message/event technology changes

Reply:
Ok - I have updated to do that. Good point.

Initially I had it like:

`await createAgreement(sbi, agreementNumber, agreement)`

But that way meant the only way I had of testing it locally was through the `async function processSubmitMessage (message, receiver) {` interface.

I think we should circle back on these at a later date and make sure the `processXXXMessage` functions are doing as little work as possible and should pass their `message.body` to another method to unpack and process, making them easier to test.

Reply:
Sounds sensible.